packages/apps-v5/src/commands/ps/index.js

```diff
@@ -31,7 +31,7 @@ function printExtended (dynos) {
 }
 
 async function printAccountQuota(context, heroku, app, account) {
-  if (app.process_tier !== 'free') {
+  if (app.process_tier !== 'free' || app.process_tier !== 'eco') {
```

Comment:
I think you want this to be `if (app.process_tier !== 'free' && app.process_tier !== 'eco') {`. Otherwise, if the `app.process_tier` is `eco`, the first check will be true and so we'll enter the block and return.

Reply:
Ahh gotcha!